server/config.js

```diff
@@ -107,6 +107,26 @@ var conf = convict({
     env: 'REDIS_DATABASE',
     format: 'int'
   },
+  db_reindex_timeout: {
+    doc: 'Number of milliseconds for etl to fetch from postgres + bulk insert into elasticsearch',
+    format: 'int',
+    default: 30000
+  },
+  db_reindex_batchSize: {
+    doc: 'Batch size for elasticsearch reindexing',
+    default: 100,
+    format: 'int'
+  },
+  db_reindex_maxRetries: {
+    doc: 'Number of times to retry an individual ETL chunk before giving up completely',
+    default: 5,
+    format: 'int'
+  },
+  db_reindex_retryWait: {
+    doc: 'Base wait period before retrying after a failure. There is a linear multiplier in the code.',
+    default: 10000,
+    format: 'int'
```

Comment:
I wonder if that Convict `duration` bug got merged and released yet, if we're interested in trying to convert these to things like "10 seconds" vs "30 seconds" vs "30 days", etc.